addons/nuxeo-retention/nuxeo-retention-core/src/main/resources/OSGI-INF/retention-schedulers.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.retention.schedulers">
+
```

Comment:
what if we rename to `<component name="org.nuxeo.retention.schedulers.contrib">` to explicitly show that we contribute instead of defining a real component.

Reply:
Our component names are such a mess that I wouldn't bother... Anyway everything in XML is technically a component, even if all it does is a contribution.

Reply:
I thought that too (using `contrib`), but then it would not be so consistent with other files, so I ended up with this one 🤷‍♂️